### PR TITLE
feat: add plugin UTRAnnotator to vep.conf (release 109)

### PR DESCRIPTION
**Pull request by diegomscoelho**
### Description

A new plugin was added to VEP_Plugins, UTRAnnotator. This is for Ensembl release e!109
Should be merged after https://github.com/Ensembl/ensembl-rest_private/pull/126 is merged.

### Use case
Plugin functionality available through REST.

### Benefits
None

### Possible Drawbacks
None

### Testing
Have you added/modified unit tests to test the changes?
Yes, I updated an old test.

If so, do the tests pass/fail?
Tests pass

Have you run the entire test suite and no regression was detected?
None detected

### Changelog
[/vep/:species/hgvs/] new plugin option added: UTRAnnotator
[/vep/:species/id/] new plugin option added: UTRAnnotator
[/vep/:species/region/] new plugin option added: UTRAnnotator

### CODE REVIEW

**Comment by nakib103** [comment]
Looks Good! It is okay to merge it.

File: root/documentation/vep.conf [hunk 1]
```diff
@@ -194,6 +194,11 @@
         description=LOFTEE identifies LoF (loss-of-function) variation. See <a target="_blank" href="https://github.com/konradjk/loftee/blob/master/README.md">README</a> for more details.
         default=0
       </LoF>
+      <UTRannotator>
+        type=Boolean
+        description=Annotates high-impact five prime UTR variants either creating new upstream ORFs or disrupting existing upstream ORFs. See <a target="_blank" href="https://raw.githubusercontent.com/ensembl-variation/VEP_plugins/main/UTRAnnotator.pm">README</a> for more details.
```
Comment (nuno-agostinho) [inline root/documentation/vep.conf:199]: To keep consistent with other plugins. 

```suggestion
        description=Annotates high-impact five prime UTR variants either creating new upstream ORFs or disrupting existing upstream ORFs (<a target="_blank" href="https://raw.githubusercontent.com/ensembl-variation/VEP_plugins/main/UTRAnnotator.pm">plugin details</a>)
```

File: root/documentation/vep.conf [hunk 2]
```diff
@@ -447,6 +452,11 @@
...
```
Comment (nuno-agostinho) [inline root/documentation/vep.conf:457]:
```suggestion
        description=Annotates high-impact five prime UTR variants either creating new upstream ORFs or disrupting existing upstream ORFs (<a target="_blank" href="https://raw.githubusercontent.com/ensembl-variation/VEP_plugins/main/UTRAnnotator.pm">plugin details</a>)
```

File: root/documentation/vep.conf [hunk 3]
```diff
@@ -687,6 +697,11 @@
...
```
Comment (nuno-agostinho) [inline root/documentation/vep.conf:702]:
```suggestion
        description=Annotates high-impact five prime UTR variants either creating new upstream ORFs or disrupting existing upstream ORFs (<a target="_blank" href="https://raw.githubusercontent.com/ensembl-variation/VEP_plugins/main/UTRAnnotator.pm">plugin details</a>)
```

File: root/documentation/vep.conf [hunk 4]
```diff
@@ -922,6 +937,11 @@
...
```
Comment (nuno-agostinho) [inline root/documentation/vep.conf:942]:
```suggestion
        description=Annotates high-impact five prime UTR variants either creating new upstream ORFs or disrupting existing upstream ORFs (<a target="_blank" href="https://raw.githubusercontent.com/ensembl-variation/VEP_plugins/main/UTRAnnotator.pm">plugin details</a>)
```

File: root/documentation/vep.conf [hunk 5]
```diff
@@ -1163,6 +1183,11 @@
...
```
Comment (nuno-agostinho) [inline root/documentation/vep.conf:1188]:
```suggestion
        description=Annotates high-impact five prime UTR variants either creating new upstream ORFs or disrupting existing upstream ORFs (<a target="_blank" href="https://raw.githubusercontent.com/ensembl-variation/VEP_plugins/main/UTRAnnotator.pm">plugin details</a>)
```

File: root/documentation/vep.conf [hunk 6]
```diff
@@ -1412,6 +1437,11 @@
...
```
Comment (nuno-agostinho) [inline root/documentation/vep.conf:1442]:
```suggestion
        description=Annotates high-impact five prime UTR variants either creating new upstream ORFs or disrupting existing upstream ORFs (<a target="_blank" href="https://raw.githubusercontent.com/ensembl-variation/VEP_plugins/main/UTRAnnotator.pm">plugin details</a>)
```

**Review by nuno-agostinho — APPROVED** [review]
Hi @diegomscoelho, I just left some minor comments to the text for consistency with other plugins.

Looks good to me!

File: root/documentation/vep.conf [hunk 7]
```diff
@@ -194,6 +194,11 @@
         description=LOFTEE identifies LoF (loss-of-function) variation. See <a target="_blank" href="https://github.com/konradjk/loftee/blob/master/README.md">README</a> for more details.
         default=0
       </LoF>
+      <UTRannotator>
```
Comment (nuno-agostinho) [inline root/documentation/vep.conf:197]: The name of the plugin needs to be fixed here and in all other instances where being used.

```suggestion
      <UTRAnnotator>
```

**Review by nuno-agostinho — CHANGES_REQUESTED** [review]

File: root/documentation/vep.conf (hunk 7 above)
Comment (diegomscoelho) [reply root/documentation/vep.conf]: Done.

**Comment by nakib103** [comment]
This change is intended for e109.

**Review by sgiorgetti — CHANGES_REQUESTED** [review]
Changes look good.
For some reasons, the test suite has failed for Perl 5.26, but it is OK for 5.14 and I cannot reproduce the failed test locally - regardless of Perl version.

There is no mention in the PR about changes to the Changelog: these are required, as we are adding in a new plugin I suppose.
May you kindly amend the PR description, accordingly?

**Comment by diegomscoelho** [comment]
Hi @sgiorgetti, I rerun web test and it seems it's working. Probably some transient error from travis.

**Comment by sgiorgetti** [comment]
> Hi @sgiorgetti, I rerun travis test and it seems it's working. Probably some transient error from travis.

Agreed

**Review by sgiorgetti — APPROVED** [review]
Changelog changes are there and build is successful.
Looks good

**Pull request merged**